tests/virt_autotest/kubevirt_tests_agent.pm

```diff
@@ -0,0 +1,91 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
```

Comment:
what's this license?

Reply:
As checked, the change was from Oliver's update. Will correct it with "Copyright 2016 SUSE LLC".

Reply:
No, keeping it just it is now should be fine.